python/tvm/relay/frontend/mxnet.py

```diff
@@ -1075,6 +1097,378 @@ def _mx_cond(inputs, attrs, subgraphs):
     return ret
 
 
+def _qnn_quantize(inputs, attrs):
+    out_dtype = 'int8'
+    out_type = attrs.get_str('out_type')
+    if out_type == 'auto':
+        if attrs.has_attr('min_calib_range') and attrs.has_attr('max_calib_range'):
+            if attrs.get_float('min_calib_range') >= 0:
+                out_dtype = 'uint8'
+            else:
+                out_dtype = 'int8'
+    else:
+        out_dtype = out_type
+    if out_dtype not in {'int8', 'uint8'}:
```

Comment:
It seems that this assert checks `out_type` actually? If that is the case, can we move it to where we obtained the `out_type` value?

Reply:
out_dtype  can be auto too. in which case we infer it based on the min and max values.